gprofiler/metadata/application_identifiers.py

```diff
@@ -68,6 +70,31 @@ def _get_cli_arg_by_name(
     return _NON_AVAILABLE_ARG
 
 
+def _get_cli_arg_by_name_multiple(
+    args: List[str], arg_name: str, check_for_equals_arg: bool = False
+) -> Optional[List[str]]:
+    results: List[str] = []
+    idx = 0
+    while True:
+        try:
+            idx = args.index(arg_name, idx)
+            results.append(args[idx + 1])
+            idx += 2
+        except ValueError:
+            break
+
+    if check_for_equals_arg:
+        for arg in args:
+            if "=" in arg:
+                arg_key, _, arg_val = arg.partition("=")
+                if arg_key == arg_name:
+                    results.append(arg_val)
+
+    if len(results) == 0:
+        return None  # returning None indicates no args found
```

Comment:
Why is it different from `_get_cli_arg_by_name` which returns `_NON_AVAILABLE_ARG`?

Reply:
:shrug: